src/pages/library/DataBrowserToggler.js

```diff
@@ -1,15 +1,83 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
 import { div, h, label, strong } from 'react-hyperscript-helpers'
-import { Link, Switch } from 'src/components/common'
+import { ButtonPrimary, IdContainer, Link, spinnerOverlay, Switch } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { TextArea, TextInput } from 'src/components/input'
+import Modal from 'src/components/Modal'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import { Ajax } from 'src/libs/ajax'
 import colors from 'src/libs/colors'
 import { isDataBrowserVisible } from 'src/libs/config'
+import { FormLabel } from 'src/libs/forms'
 import * as Nav from 'src/libs/nav'
 import { useStore } from 'src/libs/react-utils'
 import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
 import { catalogPreviewStore } from 'src/pages/library/dataBrowser-utils'
+import validate from 'validate.js'
+
+
+const DataBrowserFeedbackModal = ({ onSuccess, onDismiss }) => {
+  const [contactEmail, setContactEmail] = useState('')
```

Comment:
We talked about how we didn't want to auto-submit the user's contact info to let people send feedback anonymously, which is why it's a field vs being sent along with the feedback. Also, the email being present is an extra step a user must take to indicate that it's OK to contact them for more information. Auto-filling the field would make it too easy for a user to leave their email in place IMO.

But I could be convinced otherwise, and it's easy enough to change.

Reply:
Ah, I was assuming that it was a required field, which is why I wished it was pre-populated. I guess I didn't read the "Can we contact you with further questions?" prompt. 😅